src/plugins/orangehrmAdminPlugin/Service/LocalizationService.php

```diff
@@ -230,23 +233,26 @@ private function getXliffXmlSources(array $i18nGroups, I18NLanguage $language):
         $root->setAttribute('srcLang', 'en_US');
         $root->setAttribute('trgLang', $language->getCode());
         $root->setAttribute('xmlns', 'urn:oasis:names:tc:xliff:document:2.0');
-        $root->setAttribute('date', @date('Y-m-d\TH:i:s\Z'));
+//        $root->setAttribute('date', @date('Y-m-d\TH:i:s\Z'));
```

Comment:
date attribute is not allowed right? we can remove this line

https://docs.oasis-open.org/xliff/xliff-core/v2.0/xliff-core-v2.0.html

Reply:
removed